frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -503,8 +491,78 @@ function RunAiModelTab({ aIJobModalState }: { aIJobModalState: string }) {
   );
 }
 
+function AlignmentTab() {
+  const centerImageStyle = {
+    margin: "auto",
+    width: 150,
+  };
+  return (
+    <div>
+      <Space align="center">
+        <Radio.Button className="aIJobSelection" checked={true}>
+          <Card bordered={false}>
+            <Space direction="vertical" size="small">
+              <Row className="ai-job-title">Align Sections</Row>
+              <Row>
+                <img
+                  src={`/assets/images/${jobNameToImagePath.align_sections}`}
+                  alt={"Example of improved alignment of slices"}
+                  style={centerImageStyle}
+                />
+              </Row>
+            </Space>
+          </Card>
+        </Radio.Button>
+      </Space>
+      <AlignSectionsForm />
+    </div>
+  );
+}
+
+function UseSkeletonFormItem() {
```

Comment:
I'd prefer if we could avoid letting a component start with "Use" as this usally implies that the function is a hook and not a whole component. 

Maybe something like "ShouldUseTreesFormItem" (tree should be the more up to date name compoared to skeleton), "ShouldAlignAccordingToTreesFormItem" or something else in that direction :) The name not starting with "use" would already be helpful :pray: